src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -207,47 +214,45 @@ additional dependency options are avaialble:
 
 * **Git** dependency on a package in a folder: By default Pub assumes the
   package is located in the root of the Git repository. If that is not the
```

Comment:
I think you can reduce the "By default..." sentence and combine it with the following one.